egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -355,7 +406,7 @@ def check_for_required_files(feat_dir, tree_dir, lat_dir):
     files = ['{0}/feats.scp'.format(feat_dir), '{0}/ali.1.gz'.format(tree_dir),
              '{0}/final.mdl'.format(tree_dir), '{0}/tree'.format(tree_dir),
              '{0}/lat.1.gz'.format(lat_dir), '{0}/final.mdl'.format(lat_dir),
-             '{0}/num_jobs'.format(lat_dir), '{0}/splice_opts'.format(lat_dir)]
+             '{0}/num_jobs'.format(lat_dir)]
```

Comment:
Why is splice_opts no longer required?

Reply:
I guess it is not even required in regular chain training.

Reply:
I prefer not to remove this, though, for the sake of backwards compatibility.